packages/mui-base/src/PreviewCard/Backdrop/usePreviewCardBackdrop.types.ts

```diff
@@ -0,0 +1,7 @@
+import type { GenericHTMLProps } from '../../utils/types';
+
+export interface UsePreviewCardBackdropParameters {}
+
+export interface UsePreviewCardBackdropReturnValue {
+  getBackdropProps: (externalProps?: GenericHTMLProps) => GenericHTMLProps;
```

Comment:
Why not just name these callbacks as `getRootProps` in all hooks? No hard preference, I am just curious why we are coming up with different names in each hook. Moreover, they are all used in different component, there is no conflicts, we may not even need to destructure them under different name.

Reply:
I think it makes more sense to name it `getProps` than `getRootProps` imo (since `Root` refers to the top-level component or node, which could be confusing), but given one component's hook might render multiple nodes (I think Slider does this?), it makes more sense to add the names to each prop getter?

Reply:
> but given one component's hook might render multiple nodes (I think Slider does this?)

I was then wrong in my assumption, because I thought we are generating hook per dom node too. Yeah, feel free to leave this as is.